README-KO.md

```diff
@@ -0,0 +1,1013 @@
+
+코드는 저자에 의해 __한 번 쓰여지지만__, 많은 다른 엔지니어들에게 __반복적으로 수정되고 읽힙니다__. 대부분의 버그들은 보통 코드의 변경으로부터 나옵니다. 그래서 우리는 코드의 가독성과 유지 보수성을 향상시키기 위해 우리의 코드를 최적화 해야합니다. 이를 위한 최고의 방법은 간단한 코드를 작성하는 것입니다.
+
+Scala는 믿을 수 없을 만큼 강력하고 여러가지 페러다임을 적용 가능한 언어입니다. 우리는 아래의 가이드라인을 지키며 여러가지 프로젝트를 빠른 속도로 진행하고 있습니다. 이 가이드라인이 모든 경우에 맞지는 않을 수 있습니다. 팀이나 회사의 요구사항등에 따라서 일부 다르게 적용 해야 할 수도 있습니다.
+
+
+## <a name='TOC'>색인</a>
+
+  0. [문서 역사](#history)
+  1. [구문 스타일](#syntactic)
+    - [명명 규칙](#naming)
+    - [라인 길이](#linelength)
+    - [30 규칙](#rule_of_30)
+    - [공백 및 들여쓰기](#indent)
+    - [빈 줄](#blanklines)
+    - [괄호](#parentheses)
+    - [중괄호](#curly)
+    - [Long 정수](#long_literal)
+    - [문서 스타일](#doc)
+    - [클래스 내의 순서](#ordering_class)
+    - [Imports](#imports)
+    - [패턴 매칭](#pattern-matching)
+    - [중위 표기](#infix)
+  2. [Scala 언어의 기능](#lang)
+    - [apply 함수](#apply_method)
+    - [override 수정자](#override_modifier)
+    - [튜플 추출](#destruct_bind)
+    - [Call by Name](#call_by_name)
+    - [다중 매개 변수 표기](#multi-param-list)
+    - [특수 문자 함수 (오퍼레이터 오버라이딩)](#symbolic_methods)
+    - [타입 추론](#type_inference)
+    - [Return 예약어](#return)
+    - [재귀 용법 과 꼬리 재귀 용법](#recursion)
+    - [Implicits](#implicits)
+    - [예외 처리, i.e. )Try vs try](#exception)
+    - [Options](#option)
+    - [모나드 채이닝](#chaining)
+  3. [동시성 제어](#concurrency)
+    - [Scala concurrent.Map](#concurrency-scala-collection)
+    - [동기화(synchronized) 명시 vs Java 제공 동시성 라이브러리](#concurrency-sync-vs-map)
+    - [동기화(synchronized) 명시 vs Atomic 변수 vs @volatile](#concurrency-sync-vs-atomic)
+    - [Private 변수](#concurrency-private-this)
+    - [동시성 로직 분리](#concurrency-isolation)
+  4. [성능](#perf)
+    - [Microbenchmarks](#perf-microbenchmarks)
+    - [순회와 zipWithIndex](#perf-whileloops)
+    - [Option 과 null](#perf-option)
+    - [Scala Collection 라이브러리](#perf-collection)
+    - [private[this]](#perf-private)
+  5. [Java 호환성](#java)
+    - [Scala에서 사용 할 수 없는 Java 기능](#java-missing-features)
+    - [Traits 와 Abstract 클래스](#java-traits)
+    - [Type 별칭](#java-type-alias)
+    - [기본 매개변수 값](#java-default-param-values)
+    - [다중 매개변수 표기](#java-multi-param-list)
+    - [가변인자](#java-varargs)
+    - [Implicits](#java-implicits)
+    - [관련 객체, 정적 함수 및 변수](#java-companion-object)
+  6. [기타](#misc)
+    - [currentTimeMillis 보다는 nanoTime](#misc_currentTimeMillis_vs_nanoTime)
+    - [URL 보다는 URI](#misc_uri_url)
+
+
+
+## <a name='history'>문서 역사</a>
+- 2015-03-16: 초기 버전.
+- 2015-05-25: [override 수정자](#override_modifier) 추가.
+- 2015-08-23: "하지않습니다." 에서 "피합니다." 로 심각도 다운그레이드.
+- 2015-11-17: [apply 함수](#apply_method) 추가: apply 함수는 같은 이름을 가진 객체는 같은 이름을 가진 클레스를 반환해야 합니다.
+- 2015-11-17: 이 가이드라인이 [중국어로 번역되었습니다](README-ZH.md). 중국어 번역은 커뮤니티 맴버인 [Hawstein](https://github.com/Hawstein) 이 해주었습니다. 우리는 이 문서가 항상 최신일 것이라고 보장하지 않습니다.
+
+## <a name='syntactic'>구문 스타일</a>
+
+### <a name='naming'>명명 규칙</a>
+
+우리는 주로 Java와 Scala의 표준 명명 규칙을 따릅니다.
+
+- Class, trait, 객체는 명명규칙 즉 낙타등 표기법(PascalCase) 을 따라야 합니다.
```

Comment:
This is done but somehow it does not recognise the change, showing this comment.